scripts/tcl_commands/magic.tcl

```diff
@@ -164,38 +172,38 @@ proc run_magic_spice_export {args} {
 
 	set ::env(EXT_NETLIST) $::env(finishing_results)/$::env(DESIGN_NAME).$extract_type
 	set magic_export $::env(finishing_tmpfiles)/$extract_type.tcl
-	set commands \
-"
-if { \[info exist ::env(MAGIC_EXT_USE_GDS)\] && \$::env(MAGIC_EXT_USE_GDS) } {
-	gds read \$::env(CURRENT_GDS)
-} else {
-	lef read $::env(TECH_LEF)
-	if {  \[info exist ::env(EXTRA_LEFS)\] } {
-		set lefs_in \$::env(EXTRA_LEFS)
-		foreach lef_file \$lefs_in {
-			lef read \$lef_file
+	set commands "\
```

Comment:
how do you suggest we get rid of this pattern ?

Reply:
I have no immediate interest in doing so.